tests/testthat/test-endpoints.R

```diff
@@ -25,3 +25,22 @@ test_that("Can construct the api", {
   expect_length(logs, 2L)
   expect_identical(logs[[1L]]$logger, "daedalus.api")
 })
+
+test_that("Can get metadata", {
+  endpoint <- daedalus_api_endpoint("GET", "/metadata")
+  res <- endpoint$run()
+  expect_true(res$validated)
+  lapply(res$data$parameters$id, function(id) {
+    expect_true(id %in% expected_parameters)
+  })
```

Comment:
This could be simplified to an `expect_identical()`; if that proves too strict later it can always be changed to `expect_setequal()` or similar.
```suggestion
  expect_identical(
    res$data$parameters$id,
    expected_parameters
  )
```